### PR TITLE
Make record of resources used to release 0.28.3

### PR DESCRIPTION
**Pull request by sbwsg**
# Changes

Releasing Pipelines 0.28.3 requires older versions of some of our
resources in the dogfooding cluster. The plan is to duplicate all
release resources against a temporary namespace and run from there.

This commit includes a copy of the resources that are going to be used
for 0.28.3 so that the history of what was run is not
lost when the temp namespace is torn down after successful
release. If we need to do another point release for this version in future
hopefully this commit will help!

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Release notes block below has been filled in or deleted (only if no user facing changes)

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: tekton/build-push-ma-base-image.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: build-multiarch-base-image
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/build-push-ma-base-image.yaml:5]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/gcs-upload.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+    hub.tekton.dev/catalog: tekton
+  name: gcs-upload
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/gcs-upload.yaml:8]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/git-clone.yaml [hunk 3]
```diff
@@ -0,0 +1,107 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: git-clone
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/git-clone.yaml:5]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/golang-test.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+    hub.tekton.dev/catalog: tekton
+  name: golang-test
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/golang-test.yaml:8]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/prerelease-checks.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prerelease-checks
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/prerelease-checks.yaml:5]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/publish.yaml [hunk 6]
```diff
@@ -2,6 +2,7 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: publish-release
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/publish.yaml:5]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/release-pipeline.yaml [hunk 7]
```diff
@@ -3,6 +3,7 @@ apiVersion: tekton.dev/v1beta1
 kind: Pipeline
 metadata:
   name: pipeline-release
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/release-pipeline.yaml:6]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/release-service-account.yaml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: release-right-meow
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/release-service-account.yaml:5]:
```suggestion
  namespace: pipelines-0-28-3-release
```

File: tekton/golang-build.yaml [hunk 9]
```diff
@@ -0,0 +1,57 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+    hub.tekton.dev/catalog: tekton
+  name: golang-build
+  namespace: pipelines-0-23-8-release
```
Comment (jerop) [inline tekton/golang-build.yaml:8]:
```suggestion
  namespace: pipelines-0-28-3-release
```

**Review by jerop — COMMENTED** [review]
thanks @sbwsg 🙏🏾

**Review by jerop — APPROVED** [review]

**Comment by sbwsg** [comment]
/hold

while i update the ns here

**Comment by sbwsg** [comment]
/hold cancel

All resources should be updated to use namespace `pipelines-0-28-3-release` now.

**Comment by jerop** [comment]
thanks @sbwsg!

/lgtm

**Pull request merged**